moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -1439,4 +1574,34 @@ std::vector<ROSControlConfig> MoveItConfigData::getROSControllers()
   return ros_controllers_config_;
 }
 
-}  // namespace
\ No newline at end of file
+// Used to add a sensor plugin configuation prameter to the sensor plugin configuration parameter list
+// ******************************************************************************************
+void MoveItConfigData::addParameterToSensorPluginConfig(const std::string& name, const std::string& value,
+                                                        const std::string& comment)
+{
+  Parameter temp;
+  temp.name = name;
+  temp.value = value;
+  temp.comment = comment;
+
+  // Use index 0 since we only write one plugin
+  sensors_plugin_config_prameter_list_[0][name] = temp;
```

Comment:
if you wanna learn some cool new C++11 stuff, try bracket initialization:

```
sensors_plugin_config_prameter_list_[0][name] = {
   .name = name,
   .value = value,
   .comment = comment };
```

@mcevoyandy am i just showing off?

Reply:
Oh Thats Nice!